### ISSUE TITLE
Fire from the lights are harvestable

### ISSUE THREAD

**Issue by IdealIdeas** [mgen256/AdditionalLights#18]
You can break the fire and then pick it up and place it anywhere

**Comment by vgrynch** [comment]
This is the best feature of this mod. You can craft the fire and place it inside of your custom furniture/decoration.

**Comment by dy-inf** [comment]
Excuse me, but this is idiotic

**Comment by twenty41** [comment]
Yeah why am I able to craft and hold fire? Also creates a crafting recipe for coal alone?
